source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/organizations/settings/stores/__tests__/acceptance-test.tsx

```diff
@@ -91,7 +94,7 @@ describe('Acceptance | Organization Settings | Store view', () => {
         });
 
 
-        it('first store is selected', () => {
+        xit('first store is selected', () => {
```

Comment:
skipped tests?

Reply:
yup, the story was to temporarily make the org-settings/stores page read-only because of a lack of filtering between what stores the different organizations can use. Probably needs a data refactor to address properly.

in user testing (I think), Chris, said that it should probably be managed by the super admin for now